### DISCUSSION TITLE
[New] Proposition d'ajout de pratique : « Fournir une alternative texte aux contenus multimédias »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#24] [Bonnes pratiques] [answered]
Fournir une alternative texte aux fichiers audio et vidéo permet d'économiser de la bande passante.

C'est aussi bon pour l'accessibilité et le référencement.

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/133
